app/controllers/posts_controller.rb

```diff
@@ -25,6 +25,15 @@ def create
     end
   end
 
+  def destroy
```

Comment:
- [ ]  it would be better if you use the authorization`load_and_authorize_resource` in your controller so that it will ensure the authorization

Reply:
- Well resolved 👍🏽